### DISCUSSION TITLE
Add column MPN and #SKN in the Parts view

### DISCUSSION THREAD

**Discussion by miracuruzrh** [inventree/InvenTree#7071] [Q&A] [answered]
Hello,

I haven't found an option to add the two fields MPN and #SKN as columns to the parts view. Is it possible to customize this somewhere? My goal is to make it look like on my screenshot.

![image](https://github.com/inventree/InvenTree/assets/44341190/fb8203ce-f0e5-4103-aa17-a1f37a8a993b)

Best regards Simon

**Comment by matmair** [comment, marked as answer]
No, the relationship between parts, manufacturer parts and supplier parts are 1:n. So there would not be one SKU/MPN column but n ones.
You could write the data in metadata and add a custom view but that would be a bit counter our data model.

**Reply by miracuruzrh** [reply]
Thanks for the explanation. Sure I understand. I have not thought about that there is a 1:n relation. Of course, then it isn't possible.
I'm very new to InvenTree and not yet sure, how I should organize my inventory. It is just for my homelab, but it grew already so much with electronic parts, that I want to have a inventory for this. Therefore I try InvenTree, to see if it works for me.

Okay, I don't want to use the data model in a way it wasn't designed for. So I want to stick to, how it is intended to use.
Maybe I can ask therefore, if I'm on the right way, when I do it like in this screenshot? Adding parameters like Pin numbers and Series like in this case JST-XH and so on. And then maybe parameter like color, or type like housing, or crimp-contact or receptable or connector and so on. Or I am on the wrong way?

![image](https://github.com/inventree/InvenTree/assets/44341190/95908b95-ed1c-4860-adf4-82601232030c)

Best regards

Simon

**Reply by matmair** [reply]
This is a valid / correct use case

**Reply by miracuruzrh** [reply]
Thank you.
